Framework/Parallel/inc/MantidParallel/IO/EventLoaderHelpers.h

```diff
@@ -0,0 +1,149 @@
+#ifndef MANTID_PARALLEL_EVENTLOADERHELPERS_H_
```

Comment:
Quite a few TODOs in here. Is that deliberate?

Reply:
Re-reading the PR description I'm thinking that this comes with the parser.

Reply:
Yes.